R/pam-tidiers.R

```diff
@@ -0,0 +1,90 @@
+' Tidying methods for pam objects
```

Comment:
Lines 1- 7 will be automatically generated from the templated stuff, and can be deleted. The first line should be the `@templateVar` line.